content/sensu-go/6.2/operations/maintain-sensu/upgrade.md

```diff
@@ -11,6 +11,75 @@ menu:
     parent: maintain-sensu
 ---
 
+## Upgrade to Sensu Go 6.2.0 from any previous version
+
+Prior to Sensu Go 6.0, sensu-backend allowed you to delete a namespace even when other resources still referenced that namespace. 
+As of Sensu Go 6.0, it is not possible to delete namespaces that are referenced in other resources.
+As a result, users whose configuration predates Sensu Go 6.0 may have lingering resources, including check configurations, that reference non-existent namespaces.
+
+Upgrading to Sensu Go 6.2.0 requires sensu-backend to upgrade check configurations.
+If you have check configurations that reference non-existent namespaces, the 6.2.0 upgrade operation will fail when it encounters one of these check configurations.
+You will see an error message like this:
+
+{{< code shell >}}
+{"component":"store-providers","error":"the namespace test does not exist","level":"error","msg":"error enabling round robin scheduling,backend restart required","time":"2020-12-27T08:41:59Z"}
+{{< /code >}}
+
+When this happens, the backend is effectively halted and subsequent restarts will result in the same state.
+
+### Remove checks that reference non-existent namespaces
+
+Use the following commands with the [jq utility][7] to identify and remove checks that reference deleted namespaces before you upgrade to Sensu Go 6.2.0.
+
+{{% notice note %}}
+**NOTE**: If you have already upgraded to Sensu Go 6.2.0, you can work around this issue by temporarily reverting your Sensu instance to Sensu Go 6.1.3.
```

Comment:
Does it have to be 6.1.3? Or would it be 6.1.4 (the last release before 6.2.0) instead?

Reply:
I believe 6.1.4 would work.